tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -869,12 +870,18 @@ def _calibrate_model(self, model_name: str = None, lr: float = 0.01, max_iter: i
             if self.problem_type == BINARY:
                 y_val_probs = LabelCleanerMulticlassToBinary.convert_binary_proba_to_multiclass_proba(y_val_probs)
 
-        logger.log(15, f'Temperature scaling term being tuned for model: {model_name}')
-        temp_scalar = tune_temperature_scaling(y_val_probs=y_val_probs, y_val=y_val,
-                                               init_val=init_val, max_iter=max_iter, lr=lr)
-        logger.log(15, f'Temperature term found is: {temp_scalar}')
         model = self._trainer.load_model(model_name=model_name)
-        model.temperature_scalar = temp_scalar
+        if self.problem_type == QUANTILE:
+            logger.log(30, f'Conformity scores being computed for model: {model_name}')
```

Comment:
logging level 30 is too high (only for critical statements), set = 15 or 20 at most if you think this should be printed by default

Reply:
"Conformity scores being computed for model" 
-> 
"Conformity scores being computed to calibrate model"